modules/apps/export-import/export-import-web/src/main/java/com/liferay/exportimport/web/internal/portlet/configuration/icon/CompanyExportTemplatesConfigurationIcon.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.exportimport.web.internal.portlet.configuration.icon;
+
+import com.liferay.exportimport.constants.ExportImportPortletKeys;
+import com.liferay.portal.kernel.portlet.configuration.icon.PortletConfigurationIcon;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Carlos Correa
+ */
+@Component(
+	property = "javax.portlet.name=" + ExportImportPortletKeys.COMPANY_EXPORT,
```

Comment:
Instead of creating a base class, can't we reuse it and annotate it with both portlets like we do with the mvc commands?

Reply:
No, the component that manages these components only considers String as the value (and when we define more than one, it seems internally it is converted into an array)

Reply:
Updating that component would be required for that use case